pkg/collector/corechecks/containers/docker/events.go

```diff
@@ -97,71 +96,15 @@ func (d *DockerCheck) reportExitCodes(events []*docker.ContainerEvent, sender ag
 
 // reportEvents aggregates and sends events to the Datadog event feed
 func (d *DockerCheck) reportEvents(events []*docker.ContainerEvent, sender aggregator.Sender) error {
-	bundles := aggregateEvents(events, d.instance.FilteredEventType)
+	datadogEvs, errs := d.eventTransformer.Transform(events)
 
-	for _, bundle := range bundles {
-		ev, err := bundle.toDatadogEvent(d.dockerHostname)
-		if err != nil {
-			log.Warnf("can't submit event: %s", err)
-			continue
-		}
+	for _, err := range errs {
+		d.Warnf("Error transforming events: %s", err.Error()) //nolint:errcheck
+	}
 
+	for _, ev := range datadogEvs {
 		sender.Event(ev)
-
-		emittedEvents.Inc(string(bundle.alertType))
```

Comment:
the metric increment was removed here. Is it because it is done in another place?

Reply:
https://github.com/DataDog/datadog-agent/pull/13230/files#diff-1a910939c4b61c189278ac4914b1f6095793292a84a49507fb9040d357df6f5fR48

It is missing from the unbundled events though, I'll add it there as well.